components/dashboard-modules/NumberDisplay.tsx

```diff
@@ -7,8 +7,10 @@ interface Props {
 
 const NumberDisplay: FC<Props> = ({ title, value }) => {
   return (
-    <div className="flex-1 my-2 p-4 text-center bg-ocf-black-500 rounded-2xl">
-      <div className="mb-2 text-xs text-ocf-gray font-semibold">{title}</div>
+    <div className="flex-1 p-4 text-center md:text-left bg-ocf-gray-1000 rounded-2xl h-[100%]">
```

Comment:
This change seems to be reverting a change @ericcccsliu recently made to correct the background color of this module. Probably just left over from the merge.
```suggestion
    <div className="flex-1 p-4 text-center md:text-left bg-ocf-black-500 rounded-2xl h-[100%]">
```